amaranth-chrome-ext/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Amaranth",
+    "version": "0.1.0",
+    "manifest_version": 2
```

Comment:
What does the manifest_version do here?

Reply:
It defines which version of the manifest.json format your chrome extension follows. The most recent version is version 2, more info here: https://developer.chrome.com/extensions/manifestVersion#current-version